app/ytdl.py

```diff
@@ -58,40 +64,54 @@ def __init__(self, download_dir, output_template, output_template_chapter, quali
 
     def _download(self):
         try:
+
             def put_status(st):
-                self.status_queue.put({k: v for k, v in st.items() if k in (
-                    'tmpfilename',
-                    'filename',
-                    'status',
-                    'msg',
-                    'total_bytes',
-                    'total_bytes_estimate',
-                    'downloaded_bytes',
-                    'speed',
-                    'eta',
-                )})
+                self.status_queue.put(
+                    {
+                        k: v
+                        for k, v in st.items()
+                        if k
+                        in (

```

Comment:
I think this reformatting is not doing a good service to readability.. Is there any chance you could create a version of this PR without the reformatting?

Reply:
Sure thing. That was another auto-formatter change; I'll put it back the way it was.

Reply:
I commented on this example, but I meant the whole auto-formatter thing. I'd really rather not push the auto-formatting changes at all. Let's just push actual changes in logic

Reply:
I misunderstood; my apologies. I've reverted all formatting changes, leaving just the logic changes.